src/sass/chrome.scss

```diff
@@ -51,6 +51,7 @@ aside {
 // Header
 .pf-c-page__header {
     overflow: visible;
+    grid-template-areas: "beta beta beta" "header header header";
```

Comment:
We shouldn't need to use a grid template here

Reply:
What would you suggest using instead?